sdks/python/apache_beam/runners/job/utils.py

```diff
@@ -23,14 +23,19 @@
 from __future__ import absolute_import
 
 import json
+import logging
 
 from google.protobuf import json_format
 from google.protobuf import struct_pb2
 
 
 def dict_to_struct(dict_obj):
   # type: (dict) -> struct_pb2.Struct
-  return json_format.ParseDict(dict_obj, struct_pb2.Struct())
+  try:
+    return json_format.ParseDict(dict_obj, struct_pb2.Struct())
+  except json_format.ParseError:
+    logging.error('Failed to parse dict {0}'.format(dict_obj))
```

Comment:
lint error:
```
apache_beam/runners/job/utils.py:37:18: W1202: Use % formatting in logging functions and pass the % parameters as arguments (logging-format-interpolation)
```

Reply:
Managed to apply the change myself in the UI

Reply:
Thanks!